gitcha_test.go

```diff
@@ -30,6 +30,40 @@ func TestGitRepoForPath(t *testing.T) {
 	}
 }
 
+func TestFindAllFiles(t *testing.T) {
+	tt := []struct {
+		path string
+		list []string
+		exp  string
+	}{
+		{".", []string{"*.test"}, "ignore.test"},
+	}
+
+	for _, test := range tt {
+		_, err := os.Create(test.exp)
```

Comment:
I think it might be nicer to do this in a temp dir: https://pkg.go.dev/io/ioutil#TempDir

Reply:
even better with `t.TempDir()` https://pkg.go.dev/testing@go1.19.1#T.TempDir

Reply:
Done!